egs/gale_arabic/s5b/local/gale_format_data.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright 2014 QCRI (author: Ahmed Ali)
+# Apache 2.0
+
+if [ -f path.sh ]; then
+  . path.sh; else
+   echo "missing path.sh"; exit 1;
+fi
+
+for dir in test train; do
+   cp -pr data/local/$dir data/$dir
+done
+
+
+mkdir -p data/lang_test
+
+arpa_lm=data/local/lm/3gram-mincount/lm_unpruned.gz
+[ ! -f $arpa_lm ] && echo No such file $arpa_lm && exit 1;
+
+rm -r data/lang_test
+cp -r data/lang data/lang_test
+
+gunzip -c "$arpa_lm" | \
+  arpa2fst --disambig-symbol=#0 \
+           --read-symbol-table=data/lang_test/words.txt - data/lang_test/G.fst
+
+
+echo  "Checking how stochastic G is (the first of these numbers should be small):"
+fstisstochastic data/lang_test/G.fst
+
+## Check lexicon.
+## just have a look and make sure it seems sane.
+echo "First few lines of lexicon FST:"
+fstprint   --isymbols=data/lang/phones.txt --osymbols=data/lang/words.txt data/lang/L.fst  | head
+
+echo Performing further checks
```

Comment:
Please make sure that all echo statements of this nature start with the name of the script, e.g.:
`echo "$0: Performing further checks"`